Tests/SwiftProtobufTests/Test_AllTypes_Proto3.swift

```diff
@@ -777,13 +777,17 @@ class Test_AllTypes_Proto3: XCTestCase, PBTestHelpers {
         assertDecodeFails([253, 1, 77])
         assertDecodeFails([254, 1, 78])
         assertDecodeFails([255, 1, 79])
-        assertDebugDescription("Proto3TestAllTypes(repeatedInt32:[1])") {(o: inout MessageTestType) in
+        // WHY ISN'T THIS LIKE PROTO2?
+        //   "Proto3TestAllTypes:\nrepeated_int32: 1\n"
+        assertDebugDescription("Proto3TestAllTypes:\nrepeated_int32: [1]\n") {(o: inout MessageTestType) in
```

Comment:
> A proto3 syntax files has no control over if things are packed or not.

That would argue for not using `[...]` in the 1-item case .

> packed just controls how well behaved encoders write out the bytes, they have to be able to read both forms.

The text decoder does always support both forms, so we're already okay there.  (In fact, my work branch is dropping the packed/repeated distinction entirely in the generated decode logic, since all decoders always accept either form.)